llpc/tool/amdllpc.cpp

```diff
@@ -577,6 +577,8 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+  FinalizeSpvgen();
```

Comment:
Don't we have to finalize it in the failure case as well?

Reply:
Fixed, thanks.